e2e-pw/src/oss/specs/smoke-tests/field-visibility.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test as base } from "src/oss/fixtures";
+import { FieldVisibilityPom } from "src/oss/poms/field-visibility/field-visibility";
+import { getUniqueDatasetNameWithPrefix } from "src/oss/utils";
+import { SidebarPom } from "src/oss/poms/sidebar";
+
+const datasetName = getUniqueDatasetNameWithPrefix("smoke-quickstart");
+
+const test = base.extend<{
+  fieldVisibility: FieldVisibilityPom;
+  sidebar: SidebarPom;
+}>({
+  fieldVisibility: async ({ page }, use) => {
+    await use(new FieldVisibilityPom(page));
+  },
+  sidebar: async ({ page }, use) => {
+    await use(new SidebarPom(page));
+  },
+});
+
+test.describe("field visibility", () => {
+  test.beforeAll(async ({ fiftyoneLoader }) => {
+    await fiftyoneLoader.loadZooDataset("quickstart", datasetName, {
+      max_samples: 5,
+    });
+  });
+
+  test.beforeEach(async ({ page, fiftyoneLoader }) => {
+    await fiftyoneLoader.waitUntilGridVisible(page, datasetName);
+  });
+
+  test("field visibility modal opens", async ({ fieldVisibility }) => {
+    await fieldVisibility.openFieldVisibilityModal();
+  });
```

Comment:
there is no assertion done here

Reply:
you are right. although that test is a mock for another followup open PR. I just removed it

Reply:
I'll bring it back in https://github.com/voxel51/fiftyone/pull/3468/files